### PR TITLE
Feature/configurable logger

### PR DESCRIPTION
**Pull request by aCampello**
Description
---
Makes log buildable, i.e., every class can have it's own logs if they wish.

Closes https://github.com/wellcometrust/WellcomeML/issues/212

Checklist
---

- [x] Added link to Github issue or Trello card
- [ ] Added tests

### CODE REVIEW

File: wellcomeml/ml/bert_semantic_equivalence.py [hunk 1]
```diff
@@ -54,6 +56,7 @@ def __init__(
             random_seed: A seed used for shuffling the dataset upon training
             buffer_size: A buffer size that will be used when shuffling the dataset
         """
+        self.logger = build_logger(logging_level, __name__)
```
Comment (nsorros) [inline wellcomeml/ml/bert_semantic_equivalence.py:59]: potentially better to initialise logger outside the init

**Review by nsorros — APPROVED** [review]

File: wellcomeml/ml/bert_semantic_equivalence.py (hunk 1 above)
Comment (aCampello) [reply wellcomeml/ml/bert_semantic_equivalence.py:59]: I'm not sure where else I'd initialise it...

Comment (nsorros) [reply wellcomeml/ml/bert_semantic_equivalence.py:59]: I was thinking start of the file

**Comment by ivyleavedtoadflax** [comment]
this is nice!

**Pull request merged**